pkg/ccn-lite/ccn-lite-riot.h

```diff
@@ -12,6 +12,7 @@
 /**
  * @defgroup    pkg_ccnlite CCN-Lite stack
  * @ingroup     pkg
+ * @ingroup     net
```

Comment:
this is more a comment, than an actual change request. I will not block this effort 'cause improving documentation is always good 😉 

@miri64 what is your opinion on those additional groups?